scripts/release/xenon-survival.R

```diff
@@ -0,0 +1,109 @@
+create_survival_object <- function(){
```

Comment:
Ok, sorry for being nit-picky about this, but could you insert a space between the `)` and `{` ? (also in the rest of the code)

Reply:
Done - I appreciate aesthetics so not nit-picky at all :)